lib/membrane/rtp/header.ex

```diff
@@ -62,5 +64,13 @@ defmodule Membrane.RTP.Header do
     :timestamp,
     :sequence_number
   ]
-  defstruct @enforce_keys ++ [version: 2, marker: false, csrcs: [], extension: nil]
+  defstruct @enforce_keys ++
+              [
+                version: 2,
+                marker: false,
+                csrcs: [],
+                extension: nil,
+                has_padding?: false,
+                total_header_size: 0
```

Comment:
these were to be removed

Reply:
The reason is that I requested review before we established what to do with it :)